src/main/resources/db/changelog/changelog.sql

```diff
@@ -1061,3 +1061,13 @@ CREATE TABLE IF NOT EXISTS `Alerts` (
     CONSTRAINT `Alert-Account-Constraint` FOREIGN KEY (`userId`) REFERENCES `Accounts` (`id`) ON DELETE CASCADE ON UPDATE CASCADE,
     CONSTRAINT `Alert-Study-Constraint` FOREIGN KEY (`studyId`, `appId`) REFERENCES `Substudies` (`id`, `studyId`) ON DELETE CASCADE ON UPDATE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
+
+-- changeset bridge:77
+
+ALTER TABLE `Assessments`
+ADD `frameworkIdentifier` varchar(255) DEFAULT NULL,
+ADD `jsonSchemaUrl` varchar(500) DEFAULT NULL,
```

Comment:
@syoung-smallwisdom is a 500 char limit adequate for any conceivable jsonSchemaUrl we would want to support? I seem to remember in the early days of the Internet needing URLS that bumped up against a 1024 character limit somewhere in either the URL libraries or the web server or browser, though maybe that was just because of the site using query parameters where it should have been using a POST body.

Reply:
So far, everyone is using GitHub and 500 character limit should be fine.